scripts/ioctool

```diff
@@ -120,16 +217,49 @@ elif [ "$CMD" == "data" ]; then
 
 #################################################################
 
+elif [ "$CMD" == "autosave" ]; then
+   autosavefile=/reg/d/iocData/"${NAME}"/autosave/${NAME}.sav
+   if [ -f "$autosavefile" ]; then
+      less "$autosavefile"
+   else
+      echo "Recent autosave file could not be found."
+   fi
+
+#################################################################
+
+elif [ "$CMD" == "archive" ]; then
+   archivefile=/reg/d/iocData/"${NAME}"/archive/${NAME}.archive
+   if [ -f "$archivefile" ]; then
+      less "$archivefile"
+   else
+      echo "Archive file could not be found."
+   fi
+
+#################################################################
+
+elif [ "$CMD" == "log" ]; then
+   logfile=/reg/d/iocData/"${NAME}"/iocInfo/ioc.log
+   if [ -f "$logfile" ]; then
+      less "$logfile"
+   else
+      echo "Recent log file could not be found."
+   fi
+
+#################################################################
+
 elif [ "$CMD" == "telnet" ]; then
    INFO=$(grep_ioc "$NAME" all | grep "id:'$NAME'")
    if [ -z "$INFO" ]; then
       echo "$NAME could not be found. Exiting..." >&2
       exit 1
    fi
+   DISABLED=$(echo "$INFO" | sed -n "s/^.*disable: \(\S*\)}.*$/\1/p")
+   if [[ "$DISABLED" == "True" ]]; then
+      echo "IOC is disabled, can't connect"
+      exit 0
+   fi
```

Comment:
Up for discussion since it's a rare case but I would say not to include this part since it would take away the ability to run on iocs that are still running but shouldn't be
```suggestion
```

Reply:
It's also broken at the moment. There's a bracket instead of a comma.
```suggestion
   DISABLED=$(echo "$INFO" | sed -n "s/^.*disable: \(\S*\),.*$/\1/p")
   if [[ "$DISABLED" == "True" ]]; then
      echo "IOC is disabled, can't connect"
      exit 0
   fi
```

Reply:
How would an IOC be running when it shouldn't be? Do you mean the same ioc running in different hutches?